product-accelerators/apim/components/org.wso2.healthcare.apim.backendauth/src/main/java/org/wso2/healthcare/apim/backendauth/BackendAuthenticator.java

```diff
@@ -56,28 +63,35 @@ public boolean mediate(MessageContext messageContext) {
             log.debug("Backend authenticator mediator is invoked.");
         }
 
-        BackendAuthConfig config = backendAuthConfig.get(configName);
-        String accessToken;
-        BackendAuthHandler currentAuthenticator;
+        if (masterConfig.getClientId()!=null && masterConfig.getClientId().contains("$")){
```

Comment:
Add the formatter for the newly added code

Reply:
formatted [b459794](https://github.com/wso2/healthcare-accelerator/pull/15/commits/b4597940253b7b805fa7e95d2c2cdb0e7216d947)